server_admin/topics/clients/oidc/service-accounts.adoc

```diff
@@ -10,7 +10,11 @@ configured your <<_client-credentials, client credentials>>.
 
 To use it you must have registered a valid `confidential` Client and you need to check the switch `Service Accounts Enabled` in {project_name} admin console for this client.
 In tab `Service Account Roles` you can configure the roles available to the service account retrieved on behalf of this client.
-Don't forget that you need those roles to be available in Scopes of this client as well (unless you have `Full Scope Allowed` on). As in normal login, roles from access token are the intersection of scopes and the service account roles.
+Don't forget that you need those roles to be available in Role Scope Mappings (tab `Scope`) of this client as well (unless you
+have `Full Scope Allowed` on). As in normal login, roles from access token are the intersection of:
+
+* Role scope mappings of particular client together with the role scope mappings inherited from linked client scopes
+* Service account roles
```

Comment:
Remember that you must have the roles available in Role Scope Mappings (tab `Scope`) of this client as well, unless you
have `Full Scope Allowed` on. As in a normal login, roles from access token are the intersection of:
* Role scope mappings of particular client combined with the role scope mappings inherited from linked client scopes
* Service account roles